<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1236</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Post Process IR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="label_14">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>17</height>
       </size>
      </property>
      <property name="text">
       <string>File Selected: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="dispSelectedFile">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>601</width>
        <height>17</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>17</height>
       </size>
      </property>
      <property name="text">
       <string>Use Select File Button</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2" colspan="2">
     <layout class="QVBoxLayout" name="dispLayout"/>
    </item>
    <item row="2" column="0" rowspan="2">
     <widget class="QPlainTextEdit" name="history"/>
    </item>
    <item row="3" column="1" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QSlider" name="sl">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>739</width>
          <height>0</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="prevFrame">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Previous Frame</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="playVidBut">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pauseVidBut">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextFrame">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Next Frame</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="slStartF">
          <property name="text">
           <string>Please Select File</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="slMidF">
          <property name="text">
           <string>Please Select File</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="slEndF">
          <property name="text">
           <string>Please Select File</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="slStartT">
          <property name="text">
           <string>Please Select File</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="slMidT">
          <property name="text">
           <string>Please Select File</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="slEndT">
          <property name="text">
           <string>Please Select File</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="selectFileBut">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Select HDF5 File</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="rdoGrayScale">
          <property name="text">
           <string>GrayScale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdoIronBlack">
          <property name="text">
           <string>IronBlack</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdoRainbow">
          <property name="text">
           <string>Rainbow</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Save Frames</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Start Frame:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Stop Frame:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="startEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="stopEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hrz_Video">
        <item>
         <widget class="QPushButton" name="btnCsvVid">
          <property name="text">
           <string>CSV</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="makeTiffBut">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>TIFF</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveAsVideoSS">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>AVI</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Save Single Frame (Current Frame)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentFrameDisp">
        <property name="text">
         <string>Current Frame:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentTimeLabel">
        <property name="text">
         <string>Current Time:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hrz_SignleFrame">
        <item>
         <widget class="QPushButton" name="btnCsvPic">
          <property name="text">
           <string>CSV</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnTiffPic">
          <property name="text">
           <string>TIFF</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveCvImageBut">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>PNG</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="tempScaleBut">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Generate Figure with Temp Scale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QRadioButton" name="rdoKelvin">
          <property name="text">
           <string>Kelvin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdoCelsius">
          <property name="text">
           <string>Celsius</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdoFahrenheit">
          <property name="text">
           <string>Fahranheit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="cursorTempLabel">
        <property name="text">
         <string>Cursor Temp:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="maxTempLabel">
        <property name="text">
         <string>Current Max Temp:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="maxTempLocLabel">
        <property name="text">
         <string>Max Temp Loc:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="minTempLabel">
        <property name="text">
         <string>Current Min Temp: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="minTempLocLabel">
        <property name="text">
         <string>Min Temp Loc:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Command Log:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
